src/rpc/protocol.h

```diff
@@ -63,6 +63,9 @@ enum RPCErrorCode
     RPC_CLIENT_INVALID_IP_OR_SUBNET = -30, //!< Invalid IP/Subnet
     RPC_CLIENT_P2P_DISABLED         = -31, //!< No valid connection manager instance found
 
+    //! Chain errors
+    RPC_CLIENT_TX_POOL_DISABLED     = -33, //!< No transaction pool instance found
```

Comment:
In commit "node: Add reference to mempool in NodeContext" (fa5329abb91ec4876f61967848901c6805ab5503)

Would it be clearer to refer to the mempool instead of the tx pool or transaction pool? (I don't actually know, I'm just used to mempool over transaction pool.)

Reply:
Done. Renamed TX_POOL to MEMPOOL